umh-core/pkg/fsm/dataflowcomponent/machine.go

```diff
@@ -26,13 +26,16 @@ import (
 	"github.com/united-manufacturing-hub/united-manufacturing-hub/umh-core/pkg/constants"
 	"github.com/united-manufacturing-hub/united-manufacturing-hub/umh-core/pkg/logger"
 	"github.com/united-manufacturing-hub/united-manufacturing-hub/umh-core/pkg/metrics"
+	"github.com/united-manufacturing-hub/united-manufacturing-hub/umh-core/pkg/portmanager"
 	"github.com/united-manufacturing-hub/united-manufacturing-hub/umh-core/pkg/service/dataflowcomponent"
 )
 
 // NewDataflowComponentInstance creates a new DataflowComponentInstance with a given ID and service path
 func NewDataflowComponentInstance(
 	s6BaseDir string,
-	config config.DataFlowComponentConfig) *DataflowComponentInstance {
+	config config.DataFlowComponentConfig,
+	portManager portmanager.PortManager,
```

Comment:
for what is the port manager needed here?

Reply:
This is an unused parameter. Removed it and cleaned up the tests